tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py35,py36,py37,py38,py39}-{test,rapidjson},lint,flake8,check_package
+envlist = {py35,py36,py37,py38,py39}-{test,rapidjson},lint,flake8,check_package,black
```

Comment:
I'm fine with having static linting done _after_ testing, but changing the code after testing is just mean ;-).

Reply:
The current check doesn't change any code, it just verifies if code should be changed and if yes it prints the diff. You can locally run `black .` to apply auto-formatting.